app/src/main/java/com/example/trempel/ui/mens_category/MensCategoryFragment.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.trempel.ui.mens_category
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import androidx.recyclerview.widget.GridLayoutManager
+import com.example.trempel.MyApplication
+import com.example.trempel.databinding.MensCategoryFragmentBinding
+import javax.inject.Inject
+
+internal class MensCategoryFragment : Fragment() {
+
+    private var _binding: MensCategoryFragmentBinding? = null
+    private val binding get() = _binding!!
```

Comment:
Please, remove `!!`

`+` You shouldn't keep the reference to `binding` after `onDestroy`

Reply:
done